### PR TITLE
Dm 2312

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2312

## Description - what does this code do?
Calls javascript method to open Modal dialog designed to get feedback from users and post to survey monkey.

## Testing done - how did you test it/steps on how can another person can test it 
1.  Go to any page on the site either logged in or not.
2.  Scroll to the bottom of the page
3. click on "Send feedback" link in the footer.
4. Modal dialog should display.
image
5. Answer the questions and click on "Done" after providing your feedback.
image
6. Should see confirmation message that the data was submitted via survey monkey.
image

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
NA

## Acceptance criteria
Go to any page on site
Scroll to footer
Click "Send Feedback"
Expected: Modal appears where user can provide feedback
Current: Page just seems to refresh

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by JoshingYou1 — COMMENTED** [review]
Nice work here Brad. I do have a question though.

For the `_feedback_survey.es6` file, why can't we use the code that was already there? We could just add a class called `feedback-survey-popup` to the anchor tag. On lines `16-22`, we could keep everything except line `20`, because I don't think the focus is necessary. If we did this, we wouldn't need the `onClick` or `id` attributes for the anchor on line `19` of `_footer.html.erb`. We could also change the href to `#`, instead of `javascript:void(0)`.

**Comment by bradjohnson92008** [comment]
THanks Josh - with that class back in the main class block - it works fine.

File: spec/features/shared/footer_spec.rb [hunk 1]
```diff
@@ -27,6 +27,9 @@
         expect(page).to have_link('Nominate a practice')
       end
     end
+    it 'should open feedback modal' do
+      find('#feedback-modal').click
+    end
```
Comment (JoshingYou1) [inline spec/features/shared/footer_spec.rb:32]: Just one more thing here Brad. For this particular test, we should be expecting some sort of outcome. Can we add an expectation for the `#feedback-modal` element?

Comment (bradjohnson92008) [reply spec/features/shared/footer_spec.rb:33]: oops.. added assertion - I think I removed it.. bc the javascript wasn't being invoked with the link click.. but with fix put in yesterday - modal pops up in rspec no problem.  thx Josh

**Review by emyl3 — APPROVED** [review]
Looks good to me! Thank you, Brad!

**Pull request merged**